### PR TITLE
Add nginx 1.9.x image

### PR DESCRIPTION
**Pull request by aledbf**
```
gcr.io/google_containers/nginx-slim 0.0          038a045ad0bc   15 seconds ago  22.66 MB
gcr.io/google_containers/nginx      latest       9f4cd86c1aec   8 months ago    425.7 MB
```

### CODE REVIEW

File: Ingress/images/nginx-alpine/rc.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  selector:
+    app: nginxalpine
+  template:
+    metadata:
+      labels:
+        app: nginxalpine
+        name: frontend
+    spec:
+      containers:
+      - name: nginxalpine
```
Comment (bprashanth) [inline Ingress/images/nginx-alpine/rc.yaml]: actually lets just call this nginx-slim? that's the real goal right. if anyone really cares what it's based on they can check the docker file.

File: Ingress/images/nginx-slim/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+RUN apk add --update-cache bash && \
+  rm -rf /var/cache/apk/*
+
+COPY build.sh /tmp
+
+RUN /tmp/build.sh
+
+# forward request and error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/nginx/access.log
+RUN ln -sf /dev/stderr /var/log/nginx/error.log
```
Comment (bprashanth) [inline Ingress/images/nginx-slim/Dockerfile]: If I wrap another program that logs around this, that programs logs will show up in docker logs right? (feel like that will work out, just confirming)

File: Ingress/images/nginx-slim/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+VOLUME ["/var/cache/nginx"]
```
Comment (bprashanth) [inline Ingress/images/nginx-slim/Dockerfile]: please add a comment clarifying why we need this

**Comment by bprashanth** [comment]
LGTM but for the nits

File: Ingress/images/nginx-slim/Dockerfile (hunk 2 above)
Comment (aledbf) [inline Ingress/images/nginx-slim/Dockerfile]: @bprashanth no. Please check this https://github.com/kubernetes/contrib/pull/280/files#diff-2849bf588df199d3c10cf378cb748816R163.

Only if nginx is running in foreground (CMD or ENTRYPOINT in the dockerfile) the logs will appear. If you wrap nginx inside another process (like a `goroutine` or `cmd.Run()` in go) it doesn't work.

Comment (bprashanth) [inline Ingress/images/nginx-slim/Dockerfile]: Yes that's what i meant. Thanks for the clarification.

**Comment by aledbf** [comment]
@bprashanth done

File: Ingress/images/nginx-slim/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+COPY build.sh /tmp
+
+RUN /tmp/build.sh
+
+# Create symlinks to redirect nginx logs to stdout and stderr docker log collector
+# This only works if nginx is started with CMD or ENTRYPOINT
+RUN ln -sf /dev/stdout /var/log/nginx/access.log
+RUN ln -sf /dev/stderr /var/log/nginx/error.log
+
+VOLUME ["/var/cache/nginx"]
```
Comment (bprashanth) [inline Ingress/images/nginx-slim/Dockerfile]: comment for this volume? or did i miss it elsewhere

Comment (aledbf) [inline Ingress/images/nginx-slim/Dockerfile]: I'm sorry. I forgot to remove that.

**Comment by bprashanth** [comment]
LGTM

**Pull request merged**